packages/kbn-search-index-documents/components/result/rich_result_header.tsx

```diff
@@ -0,0 +1,238 @@
+    <strong>
+      <EuiTextColor color="subdued">{label}:</EuiTextColor>
+    </strong>
+  </EuiFlexItem>
+);
+
+const Definition: React.FC<TermDef> = ({ label }) => (
+  <EuiFlexItem>
+    <EuiTextColor color="subdued">{label}</EuiTextColor>
+  </EuiFlexItem>
+);
+const MetadataPopover: React.FC<MetaDataProps> = ({
+  id,
+  onDocumentDelete,
+  score,
+  showScore = false,
+}) => {
+  const [popoverIsOpen, setPopoverIsOpen] = useState(false);
+  const closePopover = () => setPopoverIsOpen(false);
+
+  const metaDataIcon = (
+    <EuiButtonIcon
+      display="empty"
+      size="xs"
+      iconType="iInCircle"
+      color="primary"
+      onClick={(e: React.MouseEvent<HTMLElement>) => {
+        e.stopPropagation();
+        setPopoverIsOpen(!popoverIsOpen);
+      }}
+      aria-label={i18n.translate('searchIndexDocuments.result.header.metadata.icon.ariaLabel', {
+        defaultMessage: 'Metadata for document: {id}',
+        values: { id },
+      })}
+    />
+  );
+
+  return (
+    <EuiPopover button={metaDataIcon} isOpen={popoverIsOpen} closePopover={closePopover}>
+      <EuiPopoverTitle>
+        {i18n.translate('searchIndexDocuments.result.header.metadata.title', {
+          defaultMessage: 'Document metadata',
+        })}
+      </EuiPopoverTitle>
+      <EuiFlexGroup
+        gutterSize="s"
+        direction="column"
+        css={css`
+          width: 20rem;
+        `}
+      >
+        <EuiFlexItem>
+          <EuiFlexGroup justifyContent="spaceBetween" gutterSize="s">
+            <Term label="ID" />
+            <Definition label={id} />
+          </EuiFlexGroup>
+        </EuiFlexItem>
+
+        {score && showScore && (
+          <EuiFlexItem>
+            <EuiFlexGroup justifyContent="spaceBetween" gutterSize="s">
+              <Term
+                label={i18n.translate('searchIndexDocuments.result.header.metadata.score', {
+                  defaultMessage: 'Score',
+                })}
+              />
+              <Definition label={score?.toString()} />
+            </EuiFlexGroup>
+          </EuiFlexItem>
+        )}
+      </EuiFlexGroup>
+      {onDocumentDelete && (
+        <EuiPopoverFooter>
+          <EuiButton
+            iconType="trash"
+            color="danger"
+            size="s"
+            onClick={(e: React.MouseEvent<HTMLElement>) => {
+              e.stopPropagation();
+              closePopover();
+            }}
+            fullWidth
+          >
+            {i18n.translate('searchIndexDocuments.result.header.metadata.deleteDocument', {
+              defaultMessage: 'Delete document',
+            })}
+          </EuiButton>
+        </EuiPopoverFooter>
+      )}
+    </EuiPopover>
+  );
+};
+
+const Score: React.FC<{ score: MetaDataProps['score'] }> = ({ score }) => {
+  const { euiTheme } = useEuiTheme();
+  return (
+    <EuiPanel
+      paddingSize="xs"
+      hasShadow={false}
+      color="subdued"
+      grow
+      css={css`
+        //width: ${euiTheme.base * 4}px;
```

Comment:
this a typo?

Reply:
This was supposed to go away :D it is not even correct syntax